_infra/helm/frontstage/values.yaml

```diff
@@ -32,7 +32,7 @@ ingress:
   certNameSurveys: surveys-cert
 
 analytics:
-  enabled: false
+  enabled: False
```

Comment:
not sure if this needs changing, i think i did this in error

Reply:
I've done that by mistake, good spot